packages/syntax-core/src/RichSelect/RichSelect.module.css

```diff
@@ -2,3 +2,7 @@
 .richSelectBox:focus-visible {
```

Comment:
@jliotta could you make sure to add a changeset to this PR? `pnpm run changeset`